service/src/main/java/bio/terra/landingzone/common/utils/MetricUtils.java

```diff
@@ -0,0 +1,63 @@
+package bio.terra.landingzone.common.utils;
+
+import io.micrometer.core.instrument.Metrics;
+import io.micrometer.core.instrument.Timer;
+import java.time.Duration;
+
+public class MetricUtils {
+  private static final String NAMESPACE = "landingzone";
+  private static final String CLOUD_PLATFORM_TAG = "cloudPlatform";
+  private static final String AZURE_PLATFORM_NAME = "AZURE";
+  private static final String LANDINGZONE_TYPE_TAG = "type";
+  private static final String LANDINGZONE_STEP_TAG = "step";
+
+  // this is to set reasonable upper/lower bound for a step. postgres, aks take 7-8 minutes;
+  // histogram metric would have buckets for all results.
+  private static final int LANDINGZONE_STEP_MAX_DURATION_MILLISECONDS = 60 * 10 * 1000; // 10min
```

Comment:
Nit: Why not express this as minutes in the `Duration.ofX` statements on L56-L57?

Reply:
Potentially we can reuse that specific const value, not the duration and apply duration only when it's needed. But in this case yes we can probably do it.

Reply:
I initially thought that it might be a matter of precision. But probably is not true.